context/app/static/js/components/Search/utils.js

```diff
@@ -1,12 +1,32 @@
-export function field(id, name) {
-  return {
+import fromEntries from 'fromentries';
+import searchDefinitions from '../../../../../search-schema/data/definitions.yaml';
```

Comment:
Is there a way to clean up paths to the submodules going forward instead of relative paths?